homeassistant/components/elkm1/climate.py

```diff
@@ -106,90 +112,90 @@ def target_temperature(self):
         return None
 
     @property
-    def target_temperature_high(self):
+    def target_temperature_high(self) -> float | None:
         """Return the high target temperature."""
         return self._element.cool_setpoint
 
     @property
-    def target_temperature_low(self):
+    def target_temperature_low(self) -> float | None:
         """Return the low target temperature."""
         return self._element.heat_setpoint
 
     @property
-    def target_temperature_step(self):
+    def target_temperature_step(self) -> float:
         """Return the supported step of target temperature."""
         return 1
 
     @property
-    def current_humidity(self):
+    def current_humidity(self) -> int | None:
         """Return the current humidity."""
         return self._element.humidity
 
     @property
-    def hvac_mode(self):
+    def hvac_mode(self) -> str:
         """Return current operation ie. heat, cool, idle."""
         return self._state
 
     @property
-    def hvac_modes(self):
+    def hvac_modes(self) -> list[str]:
         """Return the list of available operation modes."""
         return SUPPORT_HVAC
 
     @property
-    def precision(self):
+    def precision(self) -> int:
         """Return the precision of the system."""
         return PRECISION_WHOLE
 
     @property
-    def is_aux_heat(self):
+    def is_aux_heat(self) -> bool:
         """Return if aux heater is on."""
         return self._element.mode == ThermostatMode.EMERGENCY_HEAT.value
 
     @property
-    def min_temp(self):
+    def min_temp(self) -> float:
         """Return the minimum temperature supported."""
         return 1
 
     @property
-    def max_temp(self):
+    def max_temp(self) -> float:
         """Return the maximum temperature supported."""
         return 99
 
     @property
-    def fan_mode(self):
+    def fan_mode(self) -> str | None:
         """Return the fan setting."""
         return ELK_TO_HASS_FAN_MODES.get(self._element.fan)
```

Comment:
```suggestion
        return ELK_TO_HASS_FAN_MODES[self._element.fan]
```

Can this ever be invalid.  Do we need the `.get()` here?